app/controllers/idv/capture_doc_status_controller.rb

```diff
@@ -55,5 +60,16 @@ def throttled?
         throttle_type: :idv_doc_auth,
       ).throttled?
     end
+
+    def confirmed_barcode_attention_result?
+      flow_session[:had_barcode_attention_error] &&
+        !document_capture_session.ocr_confirmation_pending?
+    end
+
+    def pending_barcode_attention_confirmation?
+      flow_session[:had_barcode_attention_error] ||= session_result.attention_with_barcode?
```

Comment:
This might be something to improve where if a user retakes photos after a barcode attention failure, we don't want the session value to continue to be `true`. So we probably want to set it to whatever value exists for the result (`true` or `false`) as long as the result exists.